src/main/java/usi/si/seart/model/GitRepoMetric.java

```diff
@@ -0,0 +1,70 @@
+package usi.si.seart.model;
+
+import lombok.*;
```

Comment:
Do not use wildcard imports, import everything explicitly. This should be configurable somewhere in the IDE settings